crates/filament/src/ir_passes/bundle_elim.rs

```diff
@@ -4,25 +4,25 @@ use crate::{
 };
 use fil_ir::{
     self as ir, Access, AddCtx, Bind, Component, Connect, Ctx, DenseIndexInfo,
-    DisplayCtx, Expr, Foreign, InvIdx, Invoke, Liveness, MutCtx, Port, PortIdx,
-    PortOwner, Range, SparseInfoMap, Subst, Time,
+    DisplayCtx, Expr, Foreign, InvIdx, Invoke, Liveness, MutCtx, Param, Port,
+    PortIdx, PortOwner, Range, SparseInfoMap, Subst, Time,
 };
 use fil_utils as utils;
 use itertools::Itertools;
 use std::collections::HashMap;
 
-pub type PortInfo =
-    (/*lens=*/ Vec<usize>, /*gen_ports=*/ Vec<PortIdx>);
+/// A mapping from a bundle with a list of dimensions to the list of generated ports.
+type PortInfo = (/*lens=*/ Vec<usize>, /*gen_ports=*/ Vec<PortIdx>);
```

Comment:
If we're extracting these into separate types, might as well give the fields real names?